build-system/pr-check.js

```diff
@@ -207,6 +207,12 @@ function main(argv) {
     return 0;
   }
 
+  if (!(files.some(x => x === 'package.json') &&
+        files.some(x => x === 'yarn.lock'))) {
+      console.error(`\npr-check.js - please update through yarn.\n`);
```

Comment:
`pr-check.js - any update to package.json or yarn.lock must include the other file. Please update through yarn.`